### DISCUSSION TITLE
How to debug in docker

### DISCUSSION THREAD

**Discussion by N0el4kLs** [gotenberg/gotenberg#938] [Q&A]
Hi admin, is there any way to debug the code in the container, like the break point?

**Comment by gulien** [comment]
Hello @N0el4kLs,

Dunno, never try to use a Go debugger. If you find a way, please share it!

**Reply by N0el4kLs** [reply]
Sorry, I don’t understand what the meaning of ‘never try to use a Go debugger’ is. Is there anyway to debug without debugger?
